Formula/lib/libdbusmenu.rb

```diff
@@ -0,0 +1,90 @@
+class Libdbusmenu < Formula
+  desc "GLib and Gtk Implementation of the DBusMenu protocol"
+  homepage "https://launchpad.net/libdbusmenu"
+  url "https://launchpad.net/libdbusmenu/16.04/16.04.0/+download/libdbusmenu-16.04.0.tar.gz"
+  sha256 "b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a"
+  license any_of: ["GPL-3.0-only", "LGPL-2.1-or-later"]
+
+  livecheck do
+    url :homepage
+    regex(/Latest version is (\d+(?:\.\d+)+)/i)
+  end
```

Comment:
```suggestion
```
I don't think you need this; the built-in `livecheck` strategy should work fine.